apps/challenges/views.py

```diff
@@ -816,6 +817,28 @@ def create_challenge_using_zip_file(request, challenge_host_team_pk):
             response_data = {"error": message}
             return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
 
+        # To ensure that the schema for submission metaattributes is valid.
+        if (data.get("submission_meta_attributes_schema")):
+            for attribute in data["submission_meta_attributes_schema"]:
+                keys = ["name", "description", "type"]
+                missing = keys_exists_in_json(attribute, keys)
+
+                if missing == []:
```

Comment:
do a length check `len(missing) == 0` the way you did is not the right way to do it